src/include/execution/compiler/storage/pr_filler.h

```diff
@@ -0,0 +1,106 @@
+#pragma once
+#include <string>
+#include <unordered_map>
+#include <utility>
+#include "execution/compiler/function_builder.h"
+#include "execution/compiler/operator/operator_translator.h"
+#include "execution/compiler/translator_factory.h"
+
+namespace terrier::execution::compiler {
+/**
+ * PRFiller is used to generate functions that fill index PRs from table PRs
+ */
+class PRFiller : public ExpressionEvaluator {
```

Comment:
If this is an `ExpressionEvaluator`, why is it not in the `expression` sub-directory?